runtime/local/src/weights/pallet_dapps_staking.rs

```diff
@@ -2,14 +2,14 @@
 //! Autogenerated weights for `pallet_dapps_staking`
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2021-10-18, STEPS: `20`, REPEAT: 10, LOW RANGE: `[]`, HIGH RANGE: `[]`
```

Comment:
Not really important but it would be more correct to use the ones from pallet - those were measured on dev chain.

Reply:
Let's update it after, please write TODO somewhere.